src/main/java/uk/gov/hmcts/reform/orgrolemapping/data/OrganisationRefreshQueueRepository.java

```diff
@@ -28,4 +31,24 @@ public interface OrganisationRefreshQueueRepository extends JpaRepository<Organi
             + "and access_types_min_version <= :accessTypeMinVersion "
             + "and last_updated <= :lastUpdated", nativeQuery = true)
     void setActiveFalse(String organisationId, Integer accessTypeMinVersion, LocalDateTime lastUpdated);
+
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    @Modifying
+    @Query(value = "update organisation_refresh_queue "
+            + "set "
+                + "retry = case "
+                    + "when retry = 0 then 1 "
+                    + "when retry = 1 then 2 "
+                    + "when retry = 2 then 3 "
+                    + "else 4 "
+                + "end, "
+                + "retry_after = case "
+                    + "when retry = 0 then now() + (interval '1' Minute) * :retryOneInterval "
```

Comment:
maybe its just easier to pass the interval parms into the interval clause - I did that on one of my changes

Reply:
i tried that initially but kept getting error:

`Caused by: org.postgresql.util.PSQLException: ERROR: invalid input syntax for type interval: ":retryOneInterval Minute"`

i think its not possible, see SO: https://stackoverflow.com/questions/43767284/postgres-interval-not-working-with-native-spring-data-jpa-query/43767356#43767356